Tests/SourceKitLSPTests/ExecuteCommandTests.swift

```diff
@@ -46,19 +46,28 @@ final class ExecuteCommandTests: XCTestCase {
 
     let request = ExecuteCommandRequest(command: command.command, arguments: command.arguments)
 
+    let expectation = self.expectation(description: "Handle ApplyEditRequest")
+    let applyEditTitle = ThreadSafeBox<String?>(initialValue: nil)
+    let applyEditWorkspaceEdit = ThreadSafeBox<WorkspaceEdit?>(initialValue: nil)
+
     testClient.handleSingleRequest { (req: ApplyEditRequest) -> ApplyEditResponse in
+      applyEditTitle.value = req.label
+      applyEditWorkspaceEdit.value = req.edit
+      expectation.fulfill()
+
       return ApplyEditResponse(applied: true, failureReason: nil)
     }
 
-    let result = try await testClient.send(request)
+    _ = try await testClient.send(request)
```

Comment:
I think with https://github.com/swiftlang/sourcekit-lsp/pull/1519 you no longer need the `_ =`.